COPYRIGHT

```diff
@@ -0,0 +1,31 @@
+Intellectual Property Notice
+------------------------------
+
+Shroud is licensed under the BSD 3-Clause license,
+(BSD-3-Clause or https://opensource.org/licenses/BSD-3-Clause).
+
+Copyrights and patents in the Shroud project are retained by contributors.
+No copyright assignment is required to contribute to Shroud.
+
+
+SPDX usage
+------------
+
+Individual files contain SPDX tags instead of the full license text.
+This enables machine processing of license information based on the SPDX
+License Identifiers that are available here: https://spdx.org/licenses/
+
+Files that are licensed as BSD 3-Clause contain the following
+text in the license header:
+
+SPDX-License-Identifier: (BSD-3-Clause)
+
+External Packages
+-------------------
+Shroud bundles some of its external dependencies in its repository.  These
+packages are covered by various permissive licenses.  A summary listing
+follows.  See the license included with each package for full details.
+
+PackageName: fruit
```

Comment:
These need spaces at the end of the line to create a <br> linebreak.

See: https://raw.githubusercontent.com/LLNL/axom/develop/README.md

Reply:
Might only apply to ``*.md`` files.

Reply:
I moved the contents of this file over to the README.md to match axom.